### PR TITLE
[KIECLOUD-203] add feature to delete cached artifacts via build-overrides.sh

### PR DESCRIPTION
**Pull request by ricardozanini**
https://issues.jboss.org/browse/KIECLOUD-203

Signed-off-by: Ricardo Zanini <zanini@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[KIECLOUD-XYZ] Subject`, `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by ricardozanini** [comment]
I'm solving the conflicts (my fetch didn't merged with the last merge). Will force commit soon.

**Comment by ricardozanini** [comment]
Done, cherry picked from previous version

File: tools/build-overrides/build-overrides.sh [hunk 1]
```diff
@@ -613,6 +613,39 @@ EOF
...
+        fi
+    fi
+    
+    if [ "${product}" != "all" ]; then
+        query="${product}*"
+    fi
+
+    for artifact in $(cekit-cache ls | grep -B5 "${query}" | grep yaml | awk '{ print substr($0,0,length($0)-6)}'); do 
+        log_info "Deleting artifact ${artifact} from local cache"
+        #cekit-cache rm $artifact;
```
Comment (errantepiphany) [inline tools/build-overrides/build-overrides.sh]: This comment needs to be removed here like it was in https://github.com/jboss-container-images/jboss-kie-modules/pull/230/files#diff-0b96955d3c1a036fad5b60e3998a7565R645 .

Comment (ricardozanini) [reply tools/build-overrides/build-overrides.sh]: I can't believe that this passed.

**Comment by ricardozanini** [comment]
Done

**Pull request merged**